flowkit/project/program.go

```diff
@@ -42,18 +43,34 @@ func NewProgram(code []byte, args []cadence.Value, location string) (*Program, e
 	}
 
 	return &Program{
-		code:       code,
-		args:       args,
-		location:   location,
-		astProgram: astProgram,
+		code:                       code,
+		args:                       args,
+		location:                   location,
+		astProgram:                 astProgram,
+		codeWithUnprocessedImports: code, // has converted import syntax e.g. 'import "Foo"'
 	}, nil
 }
 
-// imports builds an array of all the import locations
-//
+func (p *Program) AddressImportDeclarations() []*ast.ImportDeclaration {
+	addressImports := make([]*ast.ImportDeclaration, 0)
+
+	for _, importDeclaration := range p.astProgram.ImportDeclarations() {
+		if len(importDeclaration.Identifiers) > 0 && len(importDeclaration.Location.String()) > 0 {
+			addressImports = append(addressImports, importDeclaration)
+		}
+	}
+
+	return addressImports
+}
+
+func (p *Program) HasAddressImports() bool {
+	return len(p.AddressImportDeclarations()) > 0
+}
+
+// Imports builds an array of all the import locations
 // It currently supports getting import locations as identifiers or as strings. Strings locations
 // can represent a file or an account name, whereas identifiers represent contract names.
-func (p *Program) imports() []string {
+func (p *Program) pathImports() []string {
```

Comment:
nit: I think the terminology that is used elsewhere for this type of import is "string import".  It encompasses both path imports & identifier imports (i.e. `"import X from "Y""` or `"import "X""`).

we should maybe try to stay consistent in the naming